### PR TITLE
Feat retry inline url failure

### PR DESCRIPTION
**Pull request by Karim94**
#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/285/handle-external-url-image-download-failure
#### Testing instructions  
A) Kill network while trying to load external URL image.
B) Send a link to a corrupted image.
----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: app/components/files/file-inline-image.js [hunk 1]
```diff
@@ -256,10 +294,24 @@ export default class FileInlineImage extends SafeComponent {
...
+                                <Image
+                                    onLoadStart={() => {
+                                        this.loadingTimeoutId = setTimeout(this.handleLoadingTimeout, 10000);
+                                    }}
+                                    onLoadEnd={() => {
+                                        clearTimeout(this.loadingTimeoutId);
+                                    }}
+                                    onLoad={() => {
+                                        this.loaded = true;
+                                    }}
```
Comment (seavan) [inline app/components/files/file-inline-image.js]: Let's start moving arrow functions to bound methods, instead.
Also 10000 should be moved to config, or at least to a constant.

Comment (Karim94) [reply app/components/files/file-inline-image.js]: Why should we start moving arrow functions to bound methods? Does this need to be done for all arrow functions (eventually) ?

Comment (seavan) [reply app/components/files/file-inline-image.js]: https://medium.freecodecamp.org/why-arrow-functions-and-bind-in-reacts-render-are-problematic-f1c08b060e36

**Comment by seavan** [comment]
ready for testing on a branch, @ebilg 
@Karim94 please move the task to "testing on a branch"

**Pull request merged**